tests/test_pysnooper.py

```diff
@@ -1914,3 +1916,147 @@ def f(x):
 
     with pytest.raises(TypeError):
         f()
+
+
+def test_valid_zipfile():
+    with mini_toolbox.create_temp_folder(prefix='pysnooper') as folder, \
+                                    mini_toolbox.TempSysPathAdder(str(folder)):
+        module_name = 'my_valid_zip_module'
+        zip_name = 'valid.zip'
+        zip_base_path = 'ansible/modules'
+        python_file_path = folder / ('%s/%s/%s.py' % (zip_name, zip_base_path, module_name))
+        os.makedirs(folder / ('%s/%s' % (zip_name, zip_base_path)))
+        sys.path.insert(0, str(folder / (zip_name)))
```

Comment:
Every time you insert something to `sys.path`, do a `finally` to remove it after the test. I don't want it affecting other tests.

Reply:
Also, you can remove parentheses here.

Reply:
Fair - thanks for this hint